go/vt/vtadmin/rbac/rbac.go

```diff
@@ -95,14 +95,14 @@ const (
 
 	/* generic topo resources */
 
-	CellInfoResource   Resource = "CellInfo"
-	CellsAliasResource Resource = "CellsAlias"
-	KeyspaceResource   Resource = "Keyspace"
-	ShardResource      Resource = "Shard"
-	TabletResource     Resource = "Tablet"
-	VTGateResource     Resource = "VTGate"
-	VtctldResource     Resource = "Vtctld"
-
+	CellInfoResource    Resource = "CellInfo"
+	CellsAliasResource  Resource = "CellsAlias"
+	KeyspaceResource    Resource = "Keyspace"
+	ShardResource       Resource = "Shard"
+	TabletResource      Resource = "Tablet"
+	VTGateResource      Resource = "VTGate"
+	VtctldResource      Resource = "Vtctld"
+	SrvKeyspaceResource Resource = "SrvKeyspace"
```

Comment:
two things, one extremely annoying (sorry!) and one a question i truly don't know the answer to:

1. there was a gap separating the sections that got removed; if you need to push again, mind putting it back? 😅 
2. i originally put the srv-vschema in the "vschema resources" section ... i get the pairing of "keyspace" and "srvkeyspace" in the same section, but maybe we should have a "serving graph" section that just has all the `Srv*` resources?

Reply:
Yeah that makes sense! I'll move it with the SrvVSchema resource!